Can can
bool eps_enable
bool override_ignore
uint8 eps_speed
bool acc_enable
bool aeb_enable
uint8 gear_change
uint8 turn_signal_enable
uint8 alive_count